source/runtime.cpp

```diff
@@ -873,10 +904,10 @@ namespace reshade
 		{
 			if (technique.enabled)
 			{
-				technique_list.push_back(technique.name);
+				technique_list.push_back(technique.effect_filename);
```

Comment:
Unfortunately, that's not going to work, since there are effects with multiple techniques (the reason they are grouped by filename in the first place). These two lines should be kept as they are, but in the loading routine one might check if a effect file with the same filename as the technique exists and only load those that are in the list. This still falls apart for techniques which are named different from the file, but should such a technique be encountered (which is probably not the common case), ReShade could just fall back to load everything.